From: Helge Deller <deller@gmx.de>
Date: Fri, 9 May 2014 11:46:26 +0200
Subject: Fix build on HPPA

The HPPA assembler syntax is slightly different than the GNU as to keep
compatibility to the HP-UX assembler.

Reported in https://bugs.debian.org/742797.
---
 src/bin2s.pl | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/bin2s.pl b/src/bin2s.pl
index 2c78b5e..5e2c5dc 100755
--- a/src/bin2s.pl
+++ b/src/bin2s.pl
@@ -39,7 +39,11 @@ print $ofh <<"EOF";
 
 _binary_${infile}_end:
 
+#if defined(__hppa__)
+\t_binary_${infile}_size: .equ $sz
+#else
 \t.equ _binary_${infile}_size, $sz
+#endif
 EOF
 
 close $ofh;
